packages/node/CHANGELOG.md

```diff
@@ -6,6 +6,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [3.7.0] - 2024-03-05
```

Comment:
Can we bump this version to align with the other SDK nodes please, this is behind the others